haskell/private/haskell_impl.bzl

```diff
@@ -419,7 +419,8 @@ def _create_empty_library(hs, cc, posix, my_pkg_id, with_shared, with_profiling,
 
 def haskell_library_impl(ctx):
     hs = haskell_context(ctx)
-    deps = ctx.attr.deps + ctx.attr.exports + ctx.attr.narrowed_deps
+    th_deps = getattr(ctx.attr, "th_deps", [])
```

Comment:
This looks unused.

Reply:
Isn't it used in the following line to build `deps`?